user_templates/variables.cfg

```diff
@@ -168,7 +170,8 @@ variable_material_parameters: {
             'retract_speed': 40,
             'unretract_speed': 30,
             'filter_speed': 80,
-            'additional_z_offset': 0
+            'additional_z_offset': 0,
+            'filament_sensor': 1
```

Comment:
Would it be possible to add your TPU section here to give it as an example with the sensor off?
Thanks :)

Reply:
Done.